backend/pom.xml

```diff
@@ -38,8 +38,19 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>6.1.2</version>
+        </dependency>
+		<dependency>
+			<groupId>de.flapdoodle.embed</groupId>
```

Comment:
cmd + option + l = wirkt wunder bei der formattierung :P
(nice to have, kannst du ja bei Gelegenheit nochmal nachziehen)